web/src/compose.js

```diff
@@ -161,7 +178,7 @@ export function send_message(request = create_message_object()) {
     request.resend = false;
 
     function success(data) {
-        send_message_success(local_id, data.id, locally_echoed);
+        send_message_success(request.type, local_id, data, locally_echoed);
```

Comment:
I think it might be cleaner to just pass `request, data` as parameters here.

Reply:
Hm, yeah. request and response data separate. Done.

Reply:
I think we can also now drop `local_id` and `locally_echoed`, since those are in `request`, right?

Reply:
Ah, right. Done.